src/Altinn.Auth.AuditLog.Core/Models/Partition.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Altinn.Auth.AuditLog.Core.Models
+{
+    /// <summary>
+    /// Used for partition creation
+    /// </summary>
+    public class Partition
```

Comment:
Make this a `sealed record` - gives you a better `ToString()` for debugging, and it makes sense for this class to be immutable.